src/vivliostyle/table.js

```diff
@@ -1225,19 +1350,43 @@ goog.scope(function() {
     };
 
     /**
+     * @override
```

Comment:
Fixed it in [this commit](https://github.com/vivliostyle/vivliostyle.js/commit/5624f0d56a088aba59ca3a0f5a94e5fa6b517441).